commit f895f9cedd14a3ca377b85e7a3f9213676dd3104
Author: CI Bot <ci-bot@internal.example>
Date:   Mon Feb 3 14:21:00 2025 +0000

    Add temporary rate-limiter deploy credential (TODO: move to vault)

diff --git a/deploy/rate-limiter.conf b/deploy/rate-limiter.conf
index 1a2b3c4..5d6e7f8 100644
--- a/deploy/rate-limiter.conf
+++ b/deploy/rate-limiter.conf
@@ -1,2 +1,3 @@
 name = "rate-limiter"
+secret = "S1cLhrB8quUrdZmjlC1Po1z07eouiL"
 enabled = true
